ui/00_message/achievement/achievement_name.xml

```diff
@@ -6268,14 +6295,16 @@
   <message>
     <key>ACHIEVEMENT_NAME_2546</key>
     <original>目覚めた収集家</original>
-    <translation>Collector Arisen</translation>
+    <translation>Arisen Collector</translation>
```

Comment:
Do they collect Arisen or is the Arisen a collector? I think the original reads better.